applications/neuroscan/frontend/src/components/Header.jsx

```diff
@@ -175,10 +176,17 @@ const Header = (props) => {
           intro: 'Find information about the <b>promoter</b> (primers, strains, and more) and <b>expression</b>.',
           position: 'up',
         }].filter((step) => step.element !== null),
-      }).start();
+      });
+      intro.start();
     }
   };
 
+  const handleTutorial = () => {
+    setAnchorEl(null);
+    handleViewTutorial();
+    window.document.cookie = 'introjs-dontShowAgain=true; expires=Thu, 18 Dec 2013 12:00:00 UTC';
```

Comment:
@Salam-Dalloul Imho this is not needed

Reply:
@zsinnema yes this was the issue i have talked about on slack, that this cookie needs to be removed when clicking on the hamburger menu but it wan not working with any of the solutions i have tried, and this one of them